Dockerfile.desktop

```
@@ -63,10 +74,10 @@ WORKDIR /code
 
 # Download Firefox
 ARG FIREFOX_DOWNLOAD_URL=https://download.mozilla.org/?product=firefox${FIREFOX_VERSION}-latest-ssl&lang=en-US&os=linux64
-RUN wget --no-verbose -O /tmp/firefox.tar.bz2 $FIREFOX_DOWNLOAD_URL \
+RUN wget --no-verbose -O /tmp/firefox $FIREFOX_DOWNLOAD_URL \
     && rm -rf /opt/firefox \
-    && tar -C /opt -xjf /tmp/firefox.tar.bz2 \
-    && rm /tmp/firefox.tar.bz2 \
+    && tar -C /opt -xf /tmp/firefox\
```

Not sure if it needs it, but other lines have a space before the final `\`.
```suggestion
    && tar -C /opt -xf /tmp/firefox \
```

---

> `-j` : This option tells tar to read or write archives through bzip2.

Wasn't familiar w/ all the arcane <kbd>tar</kbd> flags but story checks out since we seem to be omitting the `.tar.bz2` extension.